scylladb/sphinx-scylladb-theme
# Fix Adding a label to a heading breaks the page menu

Closes https://github.com/scylladb/sphinx-scylladb-theme/issues/613

## How to test this PR

1. Clone this pull request.

2. Run `make preview`

3. Open http://127.0.0.1:5500/examples/labels/. You should not see "Beta" next to "Usage" on the right sidebar:

![image](https://user-images.githubusercontent.com/9107969/201927663-6227398f-2144-4f1f-ba00-6f7234fd023c.png)


If you still see "Beta", please make sure to [clean your browser's cache](https://support.google.com/accounts/answer/32050?hl=en&co=GENIE.Platform%3DDesktop) and refresh the page.
